### DISCUSSION TITLE
MISSING_MESSAGE during build

### DISCUSSION THREAD

**Discussion by branislavbrincko** [amannn/next-intl#1350] [General]
next: 14.2.11 (using pages router)
next-intl: 3.19.3

When I build my app I get multiple MISSING_MESSAGE warnings (but everything is fine when the app is running). 

In `_app.tsx` I import different custom providers in which I use `useTranslations`. From what I understood the `_app` is "prebuilt" ahead of pages, in which messages are fetched, so indeed, `_app` does not have access to any messages during build. 

Is there a way how to fix this error (and are my assumptions even correct)? 

Thanks for help 🙏

**Comment by amannn** [comment]
This could be related: https://github.com/amannn/next-intl/issues/87#issuecomment-1243540265.

Otherwise have a look on the issue tracker, there's been some troubleshooting in relation to this: https://github.com/amannn/next-intl/issues?q=is%3Aissue+missing_message+is%3Aclosed+pages
